public/app/features/logs/logsModel.ts

```diff
@@ -376,7 +377,12 @@ export function logSeriesToLogsModel(
       const logsFrame = parseLogsFrame(series);
       if (logsFrame != null) {
         // for now we ignore the nested-ness of attributes, and just stringify-them
-        const frameLabels = logsFrame.getLogFrameLabelsAsLabels() ?? undefined;
+        let frameLabels = logsFrame.getLogFrameLabelsAsLabels() ?? undefined;
+        // frameLabels is undefined with SqlDataSource, but maybe other fields has be stored extraFields of LogsFrame
+        // then try to initializing it
+        if (frameLabels === undefined && logsFrame.extraFields.length > 0) {
+          frameLabels = transformToLabels(logsFrame.extraFields)
+        }
```

Comment:
I don't think this change here is correct, because we have a [defined technical specification for data frame for logs](https://grafana.com/developers/dataplane/logs) and this processing is based on that. So rather than adding this change here, we should consider adding a new possible format to mysql that would format response as logs data frame and this would work without any additional logic. 
<img width="243" alt="image" src="https://github.com/user-attachments/assets/9cf0b3af-d2bb-47b6-9b77-dd4eead06f95">

@svennergr @matyax feel free to correct me if I am wrong here